### ISSUE TITLE
Allow color palette picker 

### ISSUE THREAD

**Issue by eliorc** [arcticicestudio/nord-jetbrains#169]
When installing Nord on JetBrains' IDEs you get the standard palette for your color scheme.

In the [docs](https://www.nordtheme.com/docs/colors-and-palettes) it seems there are more beautiful palettes to be used, and I think it would be nice if they were pickable in the JetBrains' settings menus

**Comment by arcticicestudio** [comment]
Hi @eliorc 👋🏼 , thanks for your contribution 👍🏼 

I guess this is a wrong conclusion from reading the documentations.

Nord consists of 16 colors **in total**. These are divided into 4 palettes because they serve different purposes and uses cases when used for syntax highlighting or UI component design.
It's not like there are 16 different color themes. **Nord is a single theme that is made out of all these colors**. For example the colors from the _Polar Night_ palette are mostly used as background color shades for UI components while colors from the _Frost_ palette are used as main accent color for primary UI elements or to highlight functions/methods in code editors.

In short: **all colors from all palettes make up Nord**.

Although a color picker wouldn't be possible since color theme plugins "only" define color values in JSON/XML files, there is no way to implement custom components (written in Java) and extend configurations views to use and render them.
